app/js/views/services-list-item.jsx

```diff
@@ -28,6 +29,15 @@ export default class ServicesListItem extends React.Component {
           })
           .catch(console.error.bind(console));
         break;
+      case 'door-lock':
+        this.service.isLocked()
+          .then((locked) => {
+            this.setState({ locked });
+          })
+          .catch((e) => {
```

Comment:
nit: Let's avoid 1 letter variable names and use more meaningful names instead, like `error` or `err`.